src/ECharts/Event.js

```diff
@@ -1,11 +1,123 @@
+function addData(area, axis, setKey) {

```

Comment:
I do not understand what's going on in this functions, so cant really review it

Reply:
An event raised during selection of some area of chart. This event has `area` fields. These funcs adds params extracted from chart option to the event.